src/main/resources/META-INF/maven/site.vm

```diff
@@ -28,10 +28,10 @@
  | Rendered using Apache Maven Fluido Skin @project.version@
 -->
 ##
-<html xmlns="http://www.w3.org/1999/xhtml"#if ( $locale ) xml:lang="$locale.language" lang="$locale.language"#end>
```

Comment:
I am not certain about `xml:id` but dropping the namespace seems wrong. See [here](https://stackoverflow.com/a/34407786/696632) and [here](https://github.com/whatwg/html/blob/master/FAQ.md#what-is-the-namespace-declaration).

Reply:
This one was overzealous. I've put it back.